### PR TITLE
Add Support for Image based PDFs

### PR DESCRIPTION
**Pull request by arkanalexei**
This PR introduces changes by adding support for image based PDFs. This is done by adding an OCR capability into our text extractor.

### CODE REVIEW

File: rag/vectordb/postgres_node_storage.py [hunk 1]
```diff
@@ -60,9 +60,11 @@ def store_nodes(self, nodes: List[str], access_level: int):
         dimension=1536
     )
 
-    processor = TXTProcessor(DOCUMENT_PATH)
+    processor = PDFProcessor(DOCUMENT_PATH)
     nodes = processor.process()
-
+    
+    print(nodes)
```
Comment (jasonirvine76) [inline rag/vectordb/postgres_node_storage.py:66]: Do you need this print?

**Comment by nyoosteven** [comment]
Ok, I've checked your code. So you use easyocr to handle pdf based images. Do you find this interesting?

File: rag/vectordb/postgres_node_storage.py (hunk 1 above)
Comment (Insta-x) [reply rag/vectordb/postgres_node_storage.py:66]: Considering that it's just a test code not to be ran in the production code, I think it's fine.

Comment (arkanalexei) [reply rag/vectordb/postgres_node_storage.py:66]: Yes, it's just a test code

**Comment by arkanalexei** [comment]
> Ok, I've checked your code. So you use easyocr to handle pdf based images. Do you find this interesting?

What do you mean?

**Pull request merged**